src/components/MenuItem.js

```diff
@@ -82,12 +84,16 @@ const MenuItem = (props) => {
 
                 props.onPress(e);
             }}
+            onPressIn={props.onPressIn}
```

Comment:
With this change and the `onPressOut` change, we would be limiting the ability to define a different behaviour for these props in `MenuItem`. Just confirming that this is intended.

Reply:
Currently we don't need to define different onPressOut handlers for different MenuItems, so I think this is fine. If we decide to do so in the future - it will be easy to change